WheelOfFortune/Puzzle.cs

```diff
@@ -24,9 +24,9 @@ public class Puzzle
         /// Constructor initializing Puzzle instance with the input string
         /// </summary>
         /// <param name="puzzle"></param>
-        public Puzzle(string puzzle)
+        public Puzzle()
```

Comment:
I believe changes to this file are currently in master. I think you may need to pull down master, merge it into your branch, fix any merge conflicts, commit the merge, and the do a push to update the pr